python/kvikio/kvikio/buffer.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
+# See file LICENSE for terms.
+
+from kvikio._lib import buffer  # type: ignore
+
+
+def memory_register(buf) -> None:
+    """Register a device memory allocation to cuFile.
+
+    Warning
+    -------
+    This API is intended for usecases where the memory is used as streaming
```

Comment:
```suggestion
    This API is intended for usecases where the memory is used as a streaming
```

Reply:
I am not quite sure why this is annotated as a "Warning" (rather than a "Notes"), can you explain?